internal/cmd/kafka/command_cluster_update.go

```diff
@@ -56,11 +55,7 @@ func (c *clusterCommand) update(cmd *cobra.Command, args []string) error {
 	clusterID := args[0]
 	currentCluster, _, err := c.V2Client.DescribeKafkaCluster(clusterID, environmentId)
 	if err != nil {
-		environmentId, err = pconv.ConvertEnvironmentNameToId(environmentId, c.V2Client)
-		if err != nil {
-			return err
-		}
-		clusterID, err = pconv.ConvertClusterNameToId(clusterID, environmentId, c.V2Client)
+		clusterID, environmentId, err = c.clusterAndEnvNamesToIds(clusterID, environmentId)
```

Comment:
Please use consistent naming!
```suggestion
		clusterId, environmentId, err = c.clusterAndEnvNamesToIds(clusterId, environmentId)
```